docs/cdn-locations.md

```diff
@@ -0,0 +1,56 @@
+# CDN
+
+All static assets, uploaded files and cached pages generated by HM Platform are cached on a global CDN (Content Delivery Network) to ensure the best performance for website end-users. We make use of [AWS CloudFront](https://aws.amazon.com/cloudfront/features/) to provide scalable, secure and fast
+delivery of content.
```

Comment:
Remove the newline here. Had loads of issues with `<br/>`s being inserted on docs.humanmade.com with this!

Reply:
Oh oops!